starter-core/src/test/groovy/io/micronaut/starter/feature/rxjava/RxJava3Spec.groovy

```diff
@@ -42,46 +45,19 @@ class RxJava3Spec extends ApplicationContextSpec implements CommandOutputFixture
     }
 
     @Unroll
-    void 'dependency is included with maven and feature rxjava3 for language=#language'(Language language) {
+    void 'dependency is included with gradle and feature rxjava3 for language=#language'(Language language, BuildTool buildTool) {
         when:
-        String template = new BuildBuilder(beanContext, BuildTool.MAVEN)
-                .features(['rxjava3'])
+        String template = new BuildBuilder(beanContext, buildTool)
                 .language(language)
+                .features(['http-client', 'rxjava3'])
```

Comment:
If we select `['http-client-jdk', 'rxjava3']` we end up with

```
    implementation("io.micronaut:micronaut-http-client-jdk")
    implementation("io.micronaut.rxjava3:micronaut-rxjava3")
```

Which I believe is wrong...  We should probably not allow this combination?

Reply:
Or maybe we should...  So many options...